_includes/map.html

```diff
@@ -0,0 +1,38 @@
+<div id="map" class="uk-width-1-1 uk-height-large uk-margin-medium-top uk-margin-medium-bottom"></div>
```

Comment:
I'm guessing we don't really have any uses for Google Maps on this site. Is it possible to exclude the things we're not using?